tree/treeplayer/inc/TSimpleAnalysis.h

```diff
@@ -0,0 +1,81 @@
+// @(#)root/tree:$Id$
+// Author: Luca Giommi   22/08/16
+
+/*************************************************************************
+ * Copyright (C) 1995-2000, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_TSimpleAnalysis
+#define ROOT_TSimpleAnalysis
+
+//////////////////////////////////////////////////////////////////////////
+//                                                                      //
+// TSimpleAnalysis                                                      //
+//                                                                      //
+// A TSimpleAnalysis object permit to, given an input file or through   //
+// command line, create a .root file in which are saved histograms      //
+// that the user want to create.                                        //
+//                                                                      //
+//////////////////////////////////////////////////////////////////////////
+
+#ifndef ROOT_TFile
+#include "TFile.h"
+#endif
+
+#ifndef ROOT_TCanvas
+#include "TCanvas.h"
+#endif
+
+#ifndef ROOT_TChain
+#include "TChain.h"
+#endif
+
+#ifndef ROOT_TH1
+#include "TH1.h"
+#endif
+
+#ifndef ROOT_TError
+#include "TError.h"
+#endif
+
+#include <fstream>
+
+class TSimpleAnalysis {
+
+private:
+   std::string fInputName;   ///< Name of the input file
```

Comment:
Please leave only one space between `;` and `///<`.
Please indent the variable names to the same column, such that they are all aligned. I.e.
```
   std::vector<std::string> fCut; ///< Cuts added to the hisograms
   std::string              fOutputFile; ///< Output file in which are stored the histograms
```